docs/getting_started.rst

```diff
@@ -114,7 +114,29 @@ Suppose we have a transcript in CSV format containing students' basic info and t
 |     202    |     Nick   |   Young   |    Male   |  Physics  |    3.6    |
 +------------+------------+-----------+-----------+-----------+-----------+
 
-Firstly in order to set up a table, we need to specify the schema of this transcript.
+We will not include a header line in this CSV file:
+
+.. code-block:: none
+
+  200,Lucy,Smith,Female,Maths,3.8
+  200,Lucy,Smith,Female,English,3.5
+  201,Bob,King,Male,Maths,3.2
+  202,Nick,Young,Male,Physics,3.6
+
+Instead of using a header line, we will use a separate CSV config JSON file:
+
+.. code-block:: none
+
+  {
+    "header":"studentID,firstName,lastName,gender,subject,score",
+    "fileFormat":"CSV"
+  }
+
+We can create a working directory called `getting-started` on Desktop, and create two additional directories within `getting-started` called `data`
```

Comment:
Let's be more specific. 
```
$ cd $WORKING_DIR
$ mkdir getting-started
$ mkdir getting-started/data
$ mkdir getting started/config
```
and add raw data & store in `$WORKING_DIR /getting-started/data/transcript.csv`, csv config at `$WORKING_DIR/config/csv-record-reader-config.json` and so on.